index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
```

Comment:
Hi Ugboaja, please tell me where can I ask for another review? thank you

Reply:
Hi @Ugboaja-Uchechi please tell me where can I ask for another review because when I click in the button I only have the option to rate the review, thank you